mbrl/diagnostics/training_browser.py

```diff
@@ -172,12 +173,13 @@ def onExperimentsSelectionChanged(self, sel1=None, sel2=None):
                 mean_series = df.mean(axis=1)
                 var_series = df.std(axis=1)
                 
-                mean_line = self.axes.plot(time_series, mean_series)
+                mean_line = self.axes.plot(time_series, mean_series, label=self.experiment_results[rowIndex])
                 self.axes.fill_between(time_series, mean_series + var_series, mean_series - var_series, color = mean_line[0].get_color(), alpha = 0.25)
 
             if self.logYAxisCheckbox.checkState():
                 self.axes.semilogy()
 
+            self.axes.legend()
```

Comment:
This is looking great! One comment and I'm not sure if this is awkward to add. Is there any way we can have the user choose labels for the legend of the aggregated plot, and same for xlabel and ylabel?